test/test_core.py

```diff
@@ -5858,6 +5858,16 @@ def test_fs_rename_on_existing(self):
       self.set_setting('FORCE_FILESYSTEM')
     self.do_runf('fs/test_fs_rename_on_existing.c', 'success')
 
+  @parameterized({
+    '': ([],),
+    'nodefs': (['-DNODEFS', '-lnodefs.js'],),
+    'noderawfs': (['-sNODERAWFS'],)
+  })
+  def test_fs_eisdir(self, args):
+    if self.get_setting('WASMFS'):
+      self.set_setting('FORCE_FILESYSTEM')
+    self.do_runf('fs/test_fs_eisdir.c', 'success', emcc_args=args)
```

Comment:
Looks like this still need to be removed.

Reply:
Oops, fixed.